include/openenclave/internal/report.h

```diff
@@ -40,6 +41,10 @@ typedef struct _oe_get_sgx_quote_verification_collateral_args
     uint8_t* host_out_buffer;             /* out */
 } oe_get_sgx_quote_verification_collateral_args_t;
 
+// Collateral provider for sgx quote verification
+#define CRL_CA_PROCESSOR (1)
```

Comment:
Suggest using enum

Reply:
If we use enum, we have to define a new struct type here. But the parameter marshalling makes it kind of tricky.
I've added a check to ensure the value should be either 1 or 2.